packages/esm-patient-chart-app/src/workspace/workspace-window.scss

```diff
@@ -115,7 +115,7 @@
 
   .fixed {
     top: 3rem;
-    bottom: spacing.$spacing-10;
+    bottom: 3.96rem;
```

Comment:
Is there a variable we could use here instead? Why `3.96rem`? Why not `4rem`?

Reply:
It's probably better to adjust this value to `4rem` and tweak the nav height accordingly rather than use decimal values like `3.96rem` that are harder to reason about.